specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/ApplicationGroups.json

```diff
@@ -0,0 +1,327 @@
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Namespace ApplicationGroup returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationGroup"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApplicationGroupListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGroup"
+          },
+          "description": "Result of the List Application Groups operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains an incomplete list of Authorization Rules"
+        }
+      },
+      "description": "The response from the List Application Groups operation."
+    },
+    "ApplicationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "updatedAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was updated"
+            },
+            "createdAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was created."
+            },
+            "isEnabled": {
+              "type": "boolean",
+              "description": "Determines if Application Group is allowed to create connection with namespace or not. Once the isEnabled is set to false, all the existing connections of application group gets dropped and no new connections will be allowed"
+            },
+            "appGroupIdentifier": {
+              "type": "string",
+              "description": "The Unique identifier for application group.Supports SAS(SASKeyName=KeyName) or AAD(AADAppID=Guid1)"
+            },
+            "appGroupName": {
+              "type": "string",
+              "description": "The Application Group Name"
+            },
+            "policies": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ThrottlingPolicy",
+                "description": "The Application Group Policy",
+                "x-ms-client-flatten": true
```

Comment:
Remove this out of items scope. 
"description": "The Application Group Policy",
                "x-ms-client-flatten": true

Reply:
Why is this out of items scope? It is similarly done in other places.

Reply:
Removed x-ms-client-flatten property.